pandas/tests/groupby/test_groupby_shift_diff.py

```diff
@@ -154,3 +154,18 @@ def test_multindex_empty_shift_with_fill():
     shifted_with_fill = df.groupby(["a", "b"]).shift(1, fill_value=0)
     tm.assert_frame_equal(shifted, shifted_with_fill)
     tm.assert_index_equal(shifted.index, shifted_with_fill.index)
+
+
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
This is currently ignoring all future warnings. Instead, can you specify (the start of) the message that is being ignored.

Reply:
👍